src/elements/dragula/DragulaService.js

```diff
@@ -114,12 +139,16 @@ export class DragulaService {
         });
     }
 
+	/**
+     * @name setupEvents
+     * @param bag
+     */
     setupEvents(bag) {
         bag.initEvents = true;
         let that = this;
         let emitter = (type) => {
-            function replicate() {
-                let args = Array.prototype.slice.call(arguments);
+            function replicate() { // eslint-disable-line
+                let args = Array.prototype.slice.call(arguments); // eslint-disable-line
                 that[type].emit([bag.name].concat(args));
```

Comment:
This had already existed in the code. I added the comments because it wasn't related to my existing work and for some reason it was failing locally. @bvkimball is the syntax required for that library or can we refactor that?